App/CrisisCleanup/Sources/Feature/Menu/MenuViewModel.swift

```diff
@@ -64,9 +64,4 @@ class MenuViewModel: ObservableObject {
             authEventBus.onExpiredToken()
         }
     }
-}
-
-struct AccountProfilePicture {
```

Comment:
Reset this change. The `AccountProfilePicture` belongs here since it is used by this view model.